src/odemis/acq/stream/_base.py

```diff
@@ -227,7 +227,11 @@ def __init__(self, name, detector, dataflow, emitter, focuser=None, opm=None,
         self.auto_bc_outliers = model.FloatContinuous(100 / 256, range=(0, 40))
         self.auto_bc_outliers.subscribe(self._onOutliers)
 
-        self.tint = model.ListVA((255, 255, 255), unit="RGB")  # 3-int R,G,B
+        # The tint VA could be either:
+        # - a list tuple RGB value (for a tint) or
+        # - a matplotlib.colors.Colormap object for a custom color map
+        # - a string of value "fitrgb" to indicate fit RGB color mapping
+        self.tint = model.VigilantAttribute((255, 255, 255))
```

Comment:
Why don't you use a TupleContinious to also include a max value for this VA?

Reply:
So the reason is so that the VA could contain either:
- a list tuple RGB value (for a tint) or
- a matplotlib.colors.Colormap object for a custom color map
- a string of value "fitrgb" to indicate fit RGB color mapping
so it needs to be generic